dockyard/common/container/container.py

```diff
@@ -0,0 +1,162 @@
+from dockyard.common import utils, link
+
+import json
+from pecan.core import redirect
+
+class Container(object):
```

Comment:
Because this class is always going to call API having /containers therefor we can use some mechanism, where we don't have to use this string for each method. rather some variables may be defined for example class variable etc. It has to be decided.